src/components/Thumbnail.tsx

```diff
@@ -0,0 +1,96 @@
+import "./Thumbnail.scss";
+import frame85 from "./icons/Frame85.png";
+import Placeholder from "./icons/PlaceholderM.png";
+import { useIntl } from "react-intl";
+import React, { ChangeEvent, FC, useState } from "react";
+import request from "../services/request";
+
+interface ThumbnailProps {
+  propImage?: string;
+}
+
+interface imageTypes {
+  lastModified?: number;
+  lastModifiedDate?: Date;
+  name: string;
+  size: number;
+  type?: string;
+  webkitRelativePath?: string;
+}
+
+export const Thumbnail: FC<ThumbnailProps> = (propImage) => {
+  const intl = useIntl();
+  const fileReader = new FileReader();
+  const [image, setImage] = useState<imageTypes>();
+  const [imageURL, setImageURL] = useState<any>();
+  fileReader.onloadend = () => {
+    setImageURL(fileReader.result);
+  };
+  const handleOnChange = (e: ChangeEvent<any>) => {
+    e.preventDefault();
+    console.log("change", e.target.files[0]);
+    if (e.target.files[0].size <= 2000000) {
+      const file = e.target.files[0];
+      setImage(file);
+      fileReader.readAsDataURL(file);
+
+      /* const formData = new FormData();
```

Comment:
If this code is commented I think we do not need it
Please, remove it as we will spoil our code
99% that we well not uncomment it later after you merge this PR

Reply:
done